examples/seismic/abc_methods/01_introduction.ipynb

```diff
@@ -0,0 +1,1192 @@
+    "- *hx* and *hz* correspond to $\\Delta x$ and $\\Delta z$, respectively. These values are calculated in terms of the start/end points and the number of points in each direction, respectively;\n",
+    "\n",
+    "These variables are defined in the code as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "nptx   =  101\n",
+    "nptz   =  101\n",
+    "x0     =  0.\n",
+    "x1     =  1000. \n",
+    "compx  =  x1-x0\n",
+    "z0     =  0.\n",
+    "z1     =  1000.\n",
+    "compz  =  z1-z0;\n",
+    "hx     =  (x1-x0)/(nptx-1)\n",
+    "hz     =  (z1-z0)/(nptz-1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the spatial parameters are defined, we start the construction of the *spatial grid* using the Devito DSL. The required parameters are:\n",
+    "\n",
+    "- *origin* is the point of origin of the grid;\n",
+    "- *extent* is the vector with the lengths of the domain in each of the directions;\n",
+    "- *shape* is the vector with the number of points in each direction;\n",
+    "- *spacing* is the vector with the spacing in each of the directions;\n",
+    "\n",
+    "These parameters are set as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "origin  = (x0,z0)\n",
+    "extent  = (compx,compz)\n",
+    "shape   = (nptx,nptz)\n",
+    "spacing = (hx,hz)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the mesh parameters are built, we can define the structures that represent $\\Omega$ subdomains, that is, particular regions of $\\Omega $, named *subdomains*. In the present notebook, there is no need to split  the domain in particular subregions, so we define a single *subdomain* that correponds to the full domain $\\Omega$. This *subdomain* is built with the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class d0domain(SubDomain):\n",
+    "    name = 'd0'\n",
+    "    def define(self, dimensions):\n",
+    "        x, z = dimensions\n",
+    "        return {x: z, z: z}\n",
+    "d0_domain = d0domain()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After defining the spatial parameters and *subdomains*, we generate the *spatial grid*:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "grid = Grid(origin=origin, extent=extent, shape=shape, subdomains=(d0_domain))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The velocity field is set bellow, representing four different layers:"
```

Comment:
I only see 2 layers?

Reply:
You are correct. I the first's notebook there were four layers but I changed for a more simple version, using just two layers.